Description: accept newer minor versions of crate spdx
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2026-04-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -137,7 +137,7 @@
 serde_derive = "1.0.166"
 serde_json = { version = "1" }
 smallvec = "1.11.1"
-spdx = "0.10.1"
+spdx = ">= 0.10.1, <= 0.13"
 termcolor = "1.2.0"
 url = "2.0.0"
 wasmtime = { version = "34.0.1", default-features = false, features = ['cranelift', 'component-model', 'runtime', 'gc-drc'] }
